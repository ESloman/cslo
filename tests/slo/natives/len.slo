var s = "Hello world!";
print("Length of string s: ", len(s));

var x = [1, 2, 3];
print("Length of list x: ", len(x));
