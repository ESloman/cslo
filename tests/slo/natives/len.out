Length of string s: 12
Length of list x: 3
